packages/datadog_webview_tracking/example/android/detekt.yml

```diff
@@ -0,0 +1,1107 @@
+build:
```

Comment:
we also have the same file in the `datadog_flutter_plugin/example`, so maybe we can have a single file for all the projects later to avoid duplication.

Reply:
Unfortunately each package has to "stand alone" because only the files in the package directory get deployed during `pub publish`. We're having some issues where if we reference files in gradle (or other tools) that are outside the package directory it causes build failures.

So... a little bit of duplication is necessary.